File: sample/UseLocalLibrary/GNAdSamples/javaSample/src/main/java/jp/co/geniee/samples/MainActivity.java
package jp.co.geniee.samples;

import android.content.Intent;

import jp.co.geniee.samples.banner.BannerMenuActivity;
import jp.co.geniee.samples.interstitial.InterstitialDemoActivity;
import jp.co.geniee.samples.nativead.NativeAdMenuActivity;
import jp.co.geniee.samples.vast.VastDemoActivity;

public class MainActivity extends BaseMenuActivity {

    @Override
    protected MenuItem[] getListViewContents() {
        MenuItem[] menuItem = {
                new MenuItem("Banner", "Banner ads", new Intent(this, BannerMenuActivity.class)),
                new MenuItem("Native Ad", "Native advertising is the use of paid ads that match the look, feel and function of the media format in which they appear", new Intent(this, NativeAdMenuActivity.class)),
                new MenuItem("Interstitial (deprecated)", "Interstitial ads are full-screen ads that cover the interface of their host app", new Intent(this, InterstitialDemoActivity.class)),
                new MenuItem("Video Ad (deprecated)", "", new Intent(this, VastDemoActivity.class))
        };

        return menuItem;
    }
}

File: sample/UseLocalLibrary/GNAdSamples/kotlinSample/src/main/java/jp/co/geniee/samples/MainActivity.kt
package jp.co.geniee.samples

import android.content.Intent
import jp.co.geniee.samples.banner.BannerMenuActivity
import jp.co.geniee.samples.interstitial.InterstitialDemoActivity
import jp.co.geniee.samples.nativead.NativeAdMenuActivity
import jp.co.geniee.samples.vast.VastDemoActivity

class MainActivity : BaseMenuActivity() {

    override fun getListViewContents(): Array<MenuItem> {

        return arrayOf(
                MenuItem("Banner", "Banner ads", Intent(this, BannerMenuActivity::class.java)),
                MenuItem("Native Ad", "Native advertising is the use of paid ads that match the look, feel and function of the media format in which they appear", Intent(this, NativeAdMenuActivity::class.java)),
                MenuItem("Interstitial (deprecated)", "Interstitial ads are full-screen ads that cover the interface of their host app", Intent(this, InterstitialDemoActivity::class.java)),
                MenuItem("Video Ad (deprecated)", "", Intent(this, VastDemoActivity::class.java))
        )
    }
}
